libraries/apollo-ast/src/commonMain/kotlin/com/apollographql/apollo3/ast/internal/definitions.kt

```diff
@@ -15,59 +15,61 @@ internal val kotlinLabsDefinitions = """
   Since: 3.0.0
   ""${'"'}
   directive @optional(if: Boolean = true) on FIELD | VARIABLE_DEFINITION
-
+  
   ""${'"'}
   Marks a field as non-null. The corresponding Kotlin property will be made non-nullable even if the GraphQL type is nullable.
   When used on an object definition in a schema document, `fields` must be non-empty and contain a selection set of fields that should be non-null
   When used on a field from an executable document, `fields` must always be empty
-
+  
   Setting the directive at the schema level is usually easier as there is little reason that a field would be non-null in one place
   and null in the other
   Since: 3.0.0
   ""${'"'}
   directive @nonnull(fields: String! = "") on OBJECT | FIELD
-
+  
   ""${'"'}
   Attach extra information to a given type
   Since: 3.0.0
   ""${'"'}
   directive @typePolicy(
-    ""${'"'}
-    a selection set containing fields used to compute the cache key of an object. Order is important.
-    ""${'"'}
-    keyFields: String! = "",
-    ""${'"'}
-    a selection set containing fields that shouldn't create a new cache Record and should be
-    embedded in their parent instead. Order is unimportant.
-    ""${'"'}
-    embeddedFields: String! = "",
-    ""${'"'}
-    a selection set containing fields that should be treated as Relay Connection fields. Order is unimportant.
-    Since: 3.4.1
-    ""${'"'}
-    connectionFields: String! = ""
-  ) on OBJECT | INTERFACE | UNION
-
+      ""${'"'}
+      a selection set containing fields used to compute the cache key of an object. Order is important.
+      ""${'"'}
+      keyFields: String! = "",
+      ""${'"'}
+      (experimental) a selection set containing fields that shouldn't create a new cache Record and should be
+      embedded in their parent instead. Order is unimportant.
+      ""${'"'}
+      embeddedFields: String! = "",
+      ""${'"'}
+      (experimental) a selection set containing fields that should be treated as [Relay Connection](https://relay.dev/graphql/connections.htm) fields.
+      Order is unimportant.
+      This works in conjunction with `ConnectionMetadataGenerator` and `ConnectionRecordMerger` which must be configured on the `ApolloStore`.
+      Since: 3.4.1
+      ""${'"'}
+      connectionFields: String! = ""
+  ) on OBJECT | INTERFACE
```

Comment:
Looks like a new IJ thing to use the continuation indent for parameter lists. Not a huge deal but i'm hoping we can stabilize this at some point.